firebase-functions/src/main/java/com/google/firebase/functions/FirebaseFunctions.java

```diff
@@ -47,6 +50,13 @@
 /** FirebaseFunctions lets you call Cloud Functions for Firebase. */
 public class FirebaseFunctions {
 
+  /**
+   * Emulator identifier, see {@link
+   * com.google.firebase.emulators.EmulatorSettings.Builder#addEmulatedService(FirebaseEmulator,
+   * EmulatedServiceSettings)}
+   */
+  public static final FirebaseEmulator EMULATOR = FirebaseEmulator.forName("functions");
```

Comment:
Should this require a new `api.txt`?  If so the bot did not warn me for the Database PR...

Reply:
Unsure, Would it not fail tests/RC confirmation if that was the case?

Reply:
It should. Seems like the bot failed silently. See #1692

Reply:
@rosariopfernandes I'll add a fix for that here:
https://github.com/firebase/firebase-android-sdk/pull/1690

Reply:
I think I can add a fix for that here:
https://github.com/firebase/firebase-android-sdk/pull/1690

I don't think there will be a database release before that is resolved.

Reply:
@samtstern I'll change the base branch of my PR to the base branch of #1690